frontend/src/components/dashboard/geo_location/MapList.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+    <VContainer class="list-container">
+        <VSubheader
+            class="header my-3 text-dark"
+            fluid
+        >
+            Countries
+        </VSubheader>
+        <VListItem
+            v-for="item in listData"
+            :key="item.country"
+        >
+            <VListItemContent>
+                <VListItemTitle class="item-title">
+                    <div class="list-progress pl-5">
+                        <VProgressLinear
+                            :value="item.percentage"
+                            height="4"
+                            color="#7183DC"
+                        />
+                    </div>
+                    <span class="percentage pr-6">{{ item.percentage }}%</span>
+                    {{ item.country }}
+                </VListItemTitle>
+            </VListItemContent>
+        </VListItem>
+    </VContainer>
+</template>
+<script>
+    export default {
+        props: {
+            dataItems: {
+                type: Array,
+                required: true,
+            },
+            parameter: {
+                type: String,
+                required: true,
+            }
+        },
+        computed: {
+            listData () {
+                if (!this.dataItems.length) {
+                    return [];
+                }
+                let totalSum = 0;
+
+                this.dataItems.map((item) => {
```

Comment:
If you need just to loop through all items, better to use forEach, map is for transformation